gulpfile.js

```diff
@@ -22,10 +22,10 @@ var reload = browserSync.reload;
 gulp.task('browser-sync', function() {
   browserSync.init({
     server: {
-      baseDir: ["./examples", "./dev"]
+      baseDir: ["./examples", "./dev", "./dist"]
```

Comment:
Because we're building as dev, and compiled JS code is in `./dev` instead of `./dist/`